heartbeat/beater/heartbeat.go

```diff
@@ -66,11 +62,12 @@ func (bt *Heartbeat) Run(b *beat.Beat) error {
 
 	<-bt.done
 
+	bt.manager.Stop()
```

Comment:
could also be done as a defer statement on line 46 I think? Is there a reason you use `bt.manager instead of `manager`